syntax = "proto3";

package media;

service MediaService {

  rpc ExtractAudio(stream VideoChunk) returns (AudioResponse);
  rpc HealthCheck(Empty) returns (HealthStatus);
}

message Empty {}


message VideoChunk {
  bytes data = 1;
}

message AudioResponse {
  bytes audio_data = 1;
}

message HealthStatus {
  string status = 1;
}